doc/release-notes.md

```diff
@@ -28,22 +35,378 @@ reindex or re-sync the whole chain.
 
 # Notable changes
 
-## Work Queue RPC Fix / Deadlock Fix
+Masternode Reward Location Reallocation (MN_RR) Hard Fork
+---------------------------------------------------------
+The MN_RR hard fork, first included in v20 of Dash Core, will be activated after v21 is adopted by masternodes. This
+hard fork enables the major feature included in this release: Masternode Reward Location Reallocation. The activation
+will also coincide with the launch of the Dash Platform Genesis Chain.
+
+As the MN_RR hard fork is an Enhanced Hard Fork, as defined by [DIP 23](https://github.com/dashpay/dips/blob/master/dip-0023.md),
+activating is dependent on both masternodes and miners. However, as this hard fork was first implemented in v20, only
+masternodes are required to upgrade for the hard fork to activate. Once 85% of masternodes have upgraded to v21,
+an EHF message will be signed by the LLMQ_400_85, and mined into a block. This signature serves as proof that 85% of
+active masternodes have upgraded.
+
+Once the EHF signed message is mined and a cycle ends, the hard fork status will move from defined to `started`. At this point,
+miners on either v20 or v21 will begin to signal for activation of MN_RR. As such, it is likely near 100% of miners will
+be signalling. If a sufficient number of blocks mined signal for activation (starting at 80% and ticking down towards 60%)
+by the end of a cycle, the hard fork will change status from started to locked_in. At this point, hard fork activation
+will occur once another cycle ends. 
+
+Now that the hard fork is active, the new rules will be in effect.
+
+Masternode Reward Location Reallocation
+---------------------------------------
+
+Once the MN_RR hard fork activates, some coinbase funds will be moved into the Credit Pool balance (to Platform).
+Evonodes will then receive a single reward per payment cycle on the Core chain - not quad rewards (4 sequential blocks)
+as in v19/v20. The remainder of evonode payments will be distributed by Platform from the asset lock pool. This is to
+incentivize evonodes to upgrade to platform, because only nodes running platform will get these migrated rewards.
+
```

Comment:
We used both "credit pool" and "asset lock pool". Should probably stick with one.
```suggestion
Once the MN_RR hard fork activates, some coinbase funds will be moved into the Credit Pool (i.e., to Platform) each time a block is mined.
Evonodes will then receive a single reward per payment cycle on the Core chain - not rewards from four sequential blocks
as in v19/v20. The remainder of evonode payments will be distributed by Platform from the credit pool. This is to
incentivize evonodes to upgrade to Platform because only nodes running Platform can receive these reward payments.

```